frontend/packages/modelServing/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "private": true,
+  "name": "@odh-dashboard/model-serving",
+  "description": "Plugin to display model deployments. Used in combination with specific model serving platform extensions.",
+  "version": "0.0.0",
+  "exports": {
+    ".": "./src/extensions.ts",
+    "./extension-points": "./src/extension-points/index.ts"
```

Comment:
@christianvogt no way to generate this effort? We are doing `() => import('...')` seems kinda wasteful to need to do both. Future goal?

Reply:
Not exactly sure what you mean by this, but the package must export the extensions to be consumed by the main app at this time since we don't have any build plugin to do it for us.

Reply:
Definitely my mistake on what I was reading here... please ignore.